test.js

```diff
@@ -223,12 +223,20 @@ test(
 	'#999',
 	[,, '999'],
 );
-test.failing(
+test(
 	matches,
 	'forkuser#123',
 	['forkuser',, '123'],
 );
 
+// Source: As of March 2022 GitHub shows an error message when trying to create an organization with a longer name. See issue #11.
+test(
+	matches,
+	'thisorganisationnameistoolongxxxxxxxxxxx/foo#7888',
+	['foo',, '7888'],
+	'GitHub organization names can\'t be longer than 39 characters, so reference should be parsed as forkuser#number',
```

Comment:
I was initially stuck on this issue.

`IMPOSSIBLE-USERMAME/foo#123` couldn't be linkified, but the `foo#123` part now can be.

Why? The string above is equivalent to `what/ever/string/user#123`, which GitHub successfully linkifies `what/ever/string/<user#123>`